src/headers/touch.h

```diff
@@ -3,17 +3,29 @@
 
 #pragma once
 
-#define CFG_TOUCH_SETTLE 20  // Microseconds.
-#define CFG_TOUCH_TIMEOUT 20  // Microseconds.
 #define CFG_TOUCH_SMOOTH 2  // Ticks.
-#define CFG_TOUCH_DYNAMIC_MIN 3 // Microseconds
-#define CFG_TOUCH_DYNAMIC_FACTOR 2
-
-#define CFG_TOUCH_THRESHOLD_0 0  // Automatic.
-#define CFG_TOUCH_THRESHOLD_1 2  // Microseconds.
-#define CFG_TOUCH_THRESHOLD_2 3  // Microseconds.
-#define CFG_TOUCH_THRESHOLD_3 5  // Microseconds.
-#define CFG_TOUCH_THRESHOLD_4 8  // Microseconds.
+
+#define CFG_GEN0_TOUCH_TIMEOUT 20  // Microseconds.
```

Comment:
Please provide either documentation or link to documentation as to which PCB versions belong to GEN0 and to GEN1